x/poolmanager/router_test.go

```diff
@@ -36,10 +36,10 @@ type expectedTakerFees struct {
 }
 
 const (
-	foo   = "foo"
-	bar   = "bar"
-	baz   = "baz"
-	uosmo = "uosmo"
+	FOO   = apptesting.UOSMO
+	BAR   = apptesting.BAR
+	BAZ   = apptesting.BAZ
+	UOSMO = apptesting.UOSMO
```

Comment:
Note: drive by. Most diff in this file other than new test at the bottom is centered around this